File: FrontEnd/src/app/admin/Models/Inventory.ts
export class Inventory{
    public constructor(
        public flightNumber:number,
        public airline:string,
        public fromPlace:string,
        public toPlace:string,
        public startDate:any,
        public startTime:string,
        public endDate:any,
        public endTime:string, 
        public schedule:string,
        public instrumentUsed:string,
        public total_business_class_seats:number,
        public total_economy_class_seats:number,
        public ticketPrice:number,
        public no_of_rows:number,
        public mealType:string,
    ){

    }
}

File: Backend/booking/src/main/java/com/booking/model/Inventory.java
package com.booking.model;

import java.util.Date;

import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Inventory {
	
	private long inventoryId;
	private long flightNumber; 
	private Airline airline;
	private String airlineName;
	private String fromPlace;
	private String toPlace;
	private Date startDate;
	private String startTime;
	private Date endDate;
	private String endTime;
	private String schedule;
	private String instrumentUsed;
	private int total_business_class_seats;
	private int total_economy_class_seats;
	private double ticketPrice;
	private int no_of_rows;
	private String mealType;

}
